app/views/card/policy_with_member_card.html.haml

```diff
@@ -0,0 +1,33 @@
+.card-container
+    %p.logo 
+        %span.subtext They Vote For You 
+    .row
+        .col-sm-12 
+            .image-card 
+                .member-image 
+                    %img{:src => @member.large_image_url} 
+                %div.fitin
+                    .heading-text
+                        = policy_agreement_summary(@policy_person_distance, with_person: true, with_policy: true)
+    %p.disclaimer 
+        %span.subtext
+            = @link_to_policy
+
+
+
+:javascript
+    window.addEventListener('DOMContentLoaded', (event) => {
```

Comment:
I think some comments in the code here to say why this javascript code is here and what is doing would be good